integrated_channels/canvas/client.py

```diff
@@ -138,28 +141,40 @@ def create_content_metadata(self, serialized_data):
                         ),
                     )
                 )
-                status_code, response_text = self._update_course_details(
+                return self._update_course_details(
                     located_course['id'],
                     course_details,
                 )
 
-        return status_code, response_text
-
     def update_content_metadata(self, serialized_data):
         self._create_session()
 
         integration_id = self._extract_integration_id(serialized_data)
-        course_id = CanvasUtil.get_course_id_from_edx_course_id(
+        canvas_course = CanvasUtil.find_course_by_course_id(
             self.enterprise_configuration,
             self.session,
             integration_id,
         )
 
+        # If no course was found, we should create the content.
+        if not canvas_course:
+            LOGGER.info(
+                generate_formatted_log(
+                    'canvas',
```

Comment:
instead of:

`'canvas'`

try:

`self.enterprise_configuration.channel_code()`

to be more consistent with how we call this elsewhere and to make it easier to tweak these codes in the future.

Reply:
good improvement, I can also apply to the other logging here